kudosaurus.go

```diff
@@ -0,0 +1,52 @@
+package kudosaurus
+
+// Kudo - kudo entity
+type Kudo struct {
+	ID          string
+	TeamID      string
+	Text        string
+	PersonFrom  Person
+	PersonTo    Person
+	IsPublished bool
+}
+
+// Person - person entity
+type Person struct {
+	ID         string
+	Name       string
+	Kudos      []Kudo
+	KudosGiven []Kudo
+	KudosLeft  int
+	IsAdmin    bool
+}
+
+// Team - team entity
+type Team struct {
+	ID       string
+	Name     string
+	People   []Person
+	Settings Settings
+}
+
+// Settings - runtime settings for a team
+type Settings struct {
+	TeamID          string
+	TargetChannelID string
+}
+
+// Store - stores data
+type Store interface {
```

Comment:
Seems to me that this interface could be at least composed of smaller interfaces like
`PersonStore`,`KudoStore`,`TeamsStore`,`AdminStore`. This would increase readability and it's a good practice in general.